e2e/specs/st_tooltips.spec.js

```diff
@@ -79,4 +79,187 @@ describe("tooltips on widgets", () => {
     // two sliders, st.slider and st.select_slider
     cy.get(`.stSlider .stTooltipIcon`).should("have.length", 2);
   });
+
+  it("Display text properly on tooltips on text input", () => {
+    cy.get(`.stTextInput .stTooltipIcon`)
+      .invoke("show")
+      .click();
+    cy.get("[data-testid=stMarkdownContainer] .stCodeBlock").should(
+      "not.exist"
+    );
+    cy.get("[data-testid=stMarkdownContainer]").should(
+      "have.text",
+      `This is a really long tooltip.Lorem ipsum dolor sit amet, consectetur adipiscing elit. Ut ut turpis vitae\njusto ornare venenatis a vitae leo. Donec mollis ornare ante, eu ultricies\ntellus ornare eu. Donec eros risus, ultrices ut eleifend vel, auctor eu turpis.\nIn consectetur erat vel ante accumsan, a egestas urna aliquet. Nullam eget\nsapien eget diam euismod eleifend. Nulla purus enim, finibus ut velit eu,\nmalesuada dictum nulla. In non arcu et risus maximus fermentum eget nec ante.`
```

Comment:
nit (same comment for all the other checks below): would recommend factoring this out into some variables (along with the other text checked against) since it's duplicated so much, and the strings are pretty long